src/redux/types/types.js

```diff
@@ -0,0 +1,9 @@
+const BOOK_ADDED = 'BOOK_ADDED';
+const BOOK_DELETED = 'BOOK_DELETED';
+const STATUS_CHECKED = 'STATUS_CHECKED';
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md)for this project, each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), please kindly fix that.

Reply:
Thanks for the suggestion. It is to be effected